challenges/shift_array/.gitignore

```diff
@@ -0,0 +1,197 @@
+
```

Comment:
The `.gitignore` should be in the root of the repo. Any `.gitignore` not in the root will not be used.